### PR TITLE
Add new DarwinF411 targets

### PR DESCRIPTION
**Pull request by DMAXYANG**
It adds the SD card on the basis of MATEKF411, but there is a bug in the SD card of Betaflight 4.3.1.  (Fix SD card initialisation #11839)
Can you merge it to 4.3.2 as soon as possible?

### CODE REVIEW

**Comment by haslinghuis** [comment]
Targets are missing `#defines` for hardware being used.

File: configs/default/DAFP-DARWINF411.config [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+feature SOFTSERIAL
+feature TELEMETRY
+feature OSD
+
+# serial
+serial 0 64 115200 57600 0 115200
+serial 30 2048 115200 57600 0 115200
+
+# master
+set mag_hardware = NONE
```
Comment (haslinghuis) [inline configs/default/DAFP-DARWINF411.config:104]: You did specify hardware drivers to be included for mag_hardware. If this is set to NONE we should also remove mag drivers.

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/default/DAFP-DARWINF411.config (hunk 1 above)
Comment (haslinghuis) [reply configs/default/DAFP-DARWINF411.config:104]: When done please also squash your commits. Thanks.

Comment (DMAXYANG) [reply configs/default/DAFP-DARWINF411.config]: Hello, I am not familiar with Git command, please confirm whether it is correct. I feel like maybe I screwed it up

Comment (DMAXYANG) [reply configs/default/DAFP-DARWINF411.config]: I close this PR and push the new PR again

**Comment by sugaarK** [comment]
No you fix this one don’t open a new PR it’s not how we do things.. if you need help then ask

**Pull request closed without merging**